app/models/user.rb

```diff
@@ -109,15 +109,11 @@ def update_password(params)
   end
 
   # Find the users for a facility
-  # TODO: move this to facility?
   def self.find_users_by_facility(facility)
-    find_by_sql(<<-SQL)
-      SELECT u.*
-      FROM #{User.table_name} u
-      LEFT JOIN #{UserRole.table_name} ur ON u.id=ur.user_id
-      WHERE ur.facility_id = #{facility.id}
-      ORDER BY LOWER(ur.role), LOWER(u.last_name), LOWER(u.first_name)
-    SQL
+    facility
+      .users
+      .order("LOWER(user_roles.role), LOWER(users.last_name), LOWER(users.first_name)")
```

Comment:
So much nicer than the previous version. I'm still wondering if this could be turned into `Facility#users`. Does `Facility#users` get used anywhere outside this method?